internal/indexnode/task_scheduler.go

```diff
@@ -85,6 +86,24 @@ func (queue *IndexTaskQueue) addUnissuedTask(t task) error {
 	return nil
 }
 
+func (queue *IndexTaskQueue) GetUsingSlot() int64 {
```

Comment:
what about using an atomic value to record usingSlot，and modify it when task enqueued or is finished.